src/api/users/src/routes/users.js

```diff
@@ -28,29 +28,50 @@ router.get('/:id', validateId(), async (req, res, next) => {
 });
 
 // get all users
-// rejected if the user collection is empty, otherwise users returned
+// rejected if the users collection is empty
+// otherwise n (specified via params) users are returned
 router.get('/', async (req, res, next) => {
-  try {
-    const usersRef = db.collection('users');
-    const snapshot = await usersRef.get();
+  const page = !parseInt(req.query.page, 10) ? 1 : parseInt(req.query.page, 10); // if req.query.page is 0 or unparsable set to 1, otherwise set to the req.query.page
```

Comment:
Move your comments above vs. beside code, so the lines don't extend so far to the right.  Ideally we scroll down, not over when coding.